imapserver/status.go

```diff
@@ -110,12 +107,12 @@ func readStatusItem(dec *imapwire.Decoder, options *imap.StatusOptions) (isRecen
 	case "DELETED-STORAGE":
 		options.DeletedStorage = true
 	case "RECENT":
-		isRecent = true
+		options.NumRecent = true
```

Comment:
Can we return a failure if the server hasn't advertised IMAP4rev1? (ie, it's a pure IMAP4rev2 server)

Reply:
Done (handleStatus).